common/storage.go

```diff
@@ -0,0 +1,7 @@
+package common
```

Comment:
also add license header to top of files.

Reply:
fixed